### PR TITLE
Meta module update

### PR DESCRIPTION
**Pull request by petersteneteg**
Added functions to update modules to the new structure

### CODE REVIEW

File: tools/meta/include/inviwo/meta/creator.hpp [hunk 1]
```diff
@@ -56,13 +57,14 @@ class INVIWO_META_API Creator {
...
             std::optional<std::filesystem::path> templateDir = {},
-            std::optional<std::filesystem::path> configFile = {},
-            Options opts = Options{});
+            std::optional<std::filesystem::path> configFile = {}, Options opts = Options{});
 
     void createModule(const std::filesystem::path& modulePath, std::string_view org) const;
     void createFile(const std::filesystem::path& filePath) const;
     void createProcessor(const std::filesystem::path& processorPath) const;
     void createTest(const std::filesystem::path& testPath) const;
+    void updateModule(const std::filesystem::path& modulePath, std::string_view org,
```
Comment (ResearchDaniel) [inline tools/meta/include/inviwo/meta/creator.hpp:73]: What does it update?

File: tools/meta/include/inviwo/meta/includetools.hpp [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+#pragma once
+
+#include <inviwo/meta/inviwometadefine.hpp>
+
+#include <filesystem>
+#include <string_view>
+
+namespace inviwo::meta::util {
+
+INVIWO_META_API bool replaceInlcude(std::filesystem::path file, std::string_view oldInclude,
```
Comment (ResearchDaniel) [inline tools/meta/include/inviwo/meta/includetools.hpp]: mispelled -> replaceInclude
Could specify that this is only deals with C++ includes

File: tools/meta/src/inviwomoduleregistration.cpp [hunk 3]
```diff
@@ -77,7 +77,7 @@ void InviwoModuleRegistration::addInclude(const std::filesystem::path& incPath)
         res << "\n#include <" << incPath.generic_string() << ">";
         res << std::string_view{file.data() + pos, file.size() - pos};
     }
-    file = res.str();
+    file = std::move(res).str();
```
Comment (ResearchDaniel) [inline tools/meta/src/inviwomoduleregistration.cpp:80]: ?

**Review by ResearchDaniel — CHANGES_REQUESTED** [review]
Minor spelling error.

File: tools/meta/src/inviwomoduleregistration.cpp (hunk 3 above)
Comment (petersteneteg) [reply tools/meta/src/inviwomoduleregistration.cpp:80]: to allow the R-value overload of str() to steal the string buffer

**Review by ResearchDaniel — APPROVED** [review]

**Pull request merged**